packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -53,22 +53,41 @@ describe.each([
     )
   })
 
-  it("should return rows", async () => {
-    const rows = await Promise.all([
-      config.api.row.save(table._id!, { name: "foo" }),
-      config.api.row.save(table._id!, { name: "bar" }),
-    ])
+  describe("strings", () => {
+    interface StringSearchTest {
+      query: SearchFilters
+      expected: number[]
+    }
+
+    const stringSearchTests: StringSearchTest[] = [
+      { query: {}, expected: [0, 1] },
+      { query: { string: { name: "foo" } }, expected: [0] },
```

Comment:
This was not really clear on first read. Could we rename it to expectedRowIdx or similar?